bokeh/tests/selenium/README.md

```diff
@@ -0,0 +1,180 @@
+# Foreword
+
+The testing process of bokeh GUI through selenium can be done right now through two available modes:
+
+1. `Standalone` mode - you have to install bokeh and prepare full selenium
+   environment locally (then everything is started on your own computer: bokeh server and tests).
+   This mode is appropriate for local testing and writing new selenium tests. Currently this
+   mode is also used during bokeh upload to binstar-build.
+1. `Distributive` mode - most advanced approach, locally you need to install bokeh only
+   and all tests are launched in selenium grid (remotely). This allows testing through many kinds of browsers
+   as well testing through various versions of them.
+
+# Writing guidelines for adding new selenium tests
+
+1. Try to use non existent names of test cases/suites/files when creating a new test.
+   You can easily help self in this task by browsing actual names:
+
+```
+$ ./run_selenium_tests.py utils -l
+```
+
+1. Always add a short test description in each test case. Mentioned description is displayed in output of above command.
+
+# Requirements
+
+## Commmon steps (for both modes)
+
+1. Perform bokeh installation: http://bokeh.pydata.org/docs/dev_guide.html#python-setup.
+
+1. Configuring channel with testing tools:
+
+```
+$ conda config --add channels mutirri
+```
+
+Or if you want to use only packages related to selenium, then:
+
+```
+$ conda config --add channels mutirri/channel/selenium
+```
+
+1. Install some common but mandatory tools, libraries and similar stuff:
+
+```
+$ conda install selenium
+```
+
+## Additional steps for standalone mode
+
+**Note:**
+I assume that on your current system are already installed:
+
+1. Latest Chrome browser.
+1. Latest Firefox browser.
+1. Latest Opera browser.
+
+If not, please use your package manager and install all of them.
+Of course binaries of all of above browsers should be easily find on your `PATH`.
+
+### Installation of required tools
+
+1. Perform below commands:
+
+```
+$ conda install selenium-server-standalone
+$ conda install selenium-chromedriver
```

Comment:
since instructions are different for OS X for this pkg, move it below it one "if you are not on OS X do this, other wise do that" block. I almost ran this command before seeing the note about OS X below